mypy/nodes.py

```diff
@@ -448,6 +560,37 @@ def fullname(self) -> str:
     def accept(self, visitor: NodeVisitor[T]) -> T:
         return visitor.visit_var(self)
 
+    def serialize(self) -> JsonDict:
+        # TODO: Leave default values out?
+        data = {'.class': 'Var',
+                'name': self._name,
+                'fullname': self._fullname,
+                'type': None if self.type is None else self.type.serialize(),
+                'is_self': self.is_self,
+                'is_ready': self.is_ready,  # TODO: is this needed?
```

Comment:
This is not deserialized. However, for fully type checked files this should always be true.